Sources/HTTPParser/http_parser.swift

```diff
@@ -845,15 +861,15 @@ private func IS_HEADER_CHAR(_ ch: UInt8) -> Bool {
   return (ch == CR || ch == LF || ch == 9 || (ch > 31 && ch != 127))
 }
 
-private func STRICT_TOKEN(_ c: UInt8) -> UInt8 {
+private mutating func STRICT_TOKEN(_ c: UInt8) -> UInt8 {
```

Comment:
Not sure what you are trying to say here. This specific function (`STRICT_TOKEN`) isn't mutating. It is just: `return tokens[Int(c)]`. If the compiler throws an error on this, it really looks like a bug?